src/Microsoft.Health.Fhir.Core/Features/Operations/Export/ExportJobTask.cs

```diff
@@ -390,5 +409,58 @@ private async Task ProcessSearchResultsAsync(IEnumerable<SearchResultEntry> sear
                 await UpdateJobRecordAsync(cancellationToken);
             }
         }
+
+        private async Task<SearchResult> GetGroupPatients(string groupId, List<Tuple<string, string>> queryParametersList, DateTimeOffset groupMembershipTime, CancellationToken cancellationToken)
+        {
+            if (!queryParametersList.Exists((Tuple<string, string> parameter) => parameter.Item1 == KnownQueryParameterNames.Id))
```

Comment:
Should we also check whether there is a continuationtoken already present? If so, then we don't need to add the id's again right?

Reply:
I didn't realize all that information was stored in the continuation token. Good idea.